lib/isilon_api/config.rb

```diff
@@ -9,14 +9,15 @@ def self.configure()
   end
 
   class Configuration
-    attr_accessor :user, :password, :host, :port
+    attr_accessor :user, :password, :host, :port, :units
 
     def initialize
       @user     = 'example_user'
       @password = '3x4mpl3_p4$$w0rd'
       @host     = 'isilon.example.com'
       @port     = '8080'
+      @units    = 'mb'
```

Comment:
Can we allow the units to be set in the report initialization parameters instead. A global setting for a single report seems unusual here.